File: 1552-build-an-array-with-stack-operations/build-an-array-with-stack-operations.js
var buildArray = function(target, n) {
    let output = [];
    let currIndex = 0;
    for (let i = 1; i <= n; i++){
        if (i === target[currIndex]){
            output.push("Push");
            currIndex++;
        }
        else{
            output.push("Push");
            output.push("Pop");
        }
        if (i === target[target.length-1])
        break;
    }
    return output;
};

File: 1552-build-an-array-with-stack-operations/build-an-array-with-stack-operations.py
class Solution:
    def buildArray(self, target: List[int], n: int) -> List[str]:
        output = []
        currIndex = 0
        for i in range(1,n+1):
            
            if i != target[currIndex]:
                output.append("Push")
                output.append("Pop")
            else:
                output.append("Push")
                currIndex +=1
            if target[-1] == i:
                break
        return output

        